django_netjsonconfig/tests/test_admin.py

```diff
@@ -445,11 +457,13 @@ def test_context_user_not_authenticated(self):
         self.assertRedirects(response, expected_url)
 
     def test_context_vpn(self):
```

Comment:
have you tried using the decorator to patch? You can use two decorators one after the other. That would be a lot better

Reply:
I have not tried decorator to patch priorly. But, will try it out. :)